extensions/default/src/DicomWebDataSource/index.js

```diff
@@ -8,14 +8,18 @@ import {
   processResults,
   processSeriesResults,
 } from './qido.js';
-import dcm4cheeReject from './dcm4cheeReject';
 
 import getImageId from './utils/getImageId';
 import dcmjs from 'dcmjs';
 import { retrieveStudyMetadata, deleteStudyMetadataPromise } from './retrieveStudyMetadata.js';
-import StaticWadoClient from './utils/StaticWadoClient';
 import getDirectURL from '../utils/getDirectURL';
 import { fixBulkDataURI } from './utils/fixBulkDataURI';
+import {
+  mergedSearch,
```

Comment:
Please get rid of mergedSearch etc.  You  don't need to merge the search, simply perform each query against the named data source, providing the RetrieveAE in the keyed value.